### PR TITLE
Curve hourly snapshot

### PR DESCRIPTION
**Pull request by sevenshi**

### CODE REVIEW

**Comment by this-username-is-taken** [comment]
Is this ready or still a work in progress?

File: subgraphs/curve-finance/src/services/pools.ts [hunk 1]
```diff
@@ -70,7 +70,8 @@ export function createNewPool(
   pool.gauge = gaugeAddress.toHexString();
   const inputTokens = getPoolCoins(pool);
   pool.coins = inputTokens;
-  pool.inputTokens = inputTokens.sort();
+  pool.inputTokens = inputTokens;
```
Comment (this-username-is-taken) [inline subgraphs/curve-finance/src/services/pools.ts:73]: Is this the only change? If so, can you revert all the other changes (including the commented line below)?

Comment (this-username-is-taken) [inline subgraphs/curve-finance/src/services/pools.ts:73]: Is this the only change? If so, can you revert all the other changes (including the commented line below)?

Comment (sevenshi) [reply subgraphs/curve-finance/src/services/pools.ts:73]: yes  ok

**Pull request closed without merging**